test/functional/test_framework/address.py

```diff
@@ -41,7 +44,32 @@ def byte_to_base58(b, version):
         str = str[2:]
     return result
 
-# TODO: def base58_decode
+
+def base58_to_byte(s, version, verify_checksum=True):
```

Comment:
In commit "tests: implement base58_decode" (31cfd81e1f374a722ab0190cd89814f22b8f91f8)

I think version argument is confusing and should be dropped. There's no need to require it since version number is in res[0]